alkira/resource_alkira_connector_internet_exit.go

```diff
@@ -172,6 +182,7 @@ func generateConnectorInternetRequest(d *schema.ResourceData, m interface{}) (*a
 	client := m.(*alkira.AlkiraClient)
 
 	billingTags := convertTypeListToIntList(d.Get("billing_tag_ids").([]interface{}))
+	egressTypes := convertTypeListToStringList(d.Get("egress_ips").([]interface{}))
```

Comment:
You will also need to add this to the `resourceConnectorInternetExitRead` function